content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -722,6 +722,92 @@ controller is tracking a Job using Pod finalizers by checking if the Job has the
 annotation `batch.kubernetes.io/job-tracking`. You should **not** manually add
 or remove this annotation from Jobs.
 
+### Specifying your Pod failure policy
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+In some situations you may want to have a better control over handling of Pod
+failures than the default policy based on the Job's `.spec.backoffLimit` gives
+(see: [Pod backoff failure policy](#pod-backoff-failure-policy)).
+
+For example, you may want to terminate a job as soon as one of its Pods fails
+with an exit code indicating a software bug (for example due to an unknown
+command invoked). This may allow you to cut down costs of running computational
+workloads by avoiding unnecessary Pod restarts. Additionally, you may want to do
+not count pod failures caused by disruptions (such as preemption, eviction API
+or taint-based Pod deletions) towards the `.spec.backoffLimit` counter.
+
+You can satisfy the above use cases by specifying a Pod failure policy
+(in the `.spec.podFailrePolicy` field) for handling Pod failures based on the
+container exit codes and the Pod conditions.
+
+{{< note >}}
+You can only configure a Pod failure policy for a Job if you have the
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster: `JobPodFailurePolicy`. Additionally, it is recommended
+to enable the `PodDisruptionsCondition` in order to be able to detect and handle
+pod disruption conditions in the Pod failure policy. Both feature gates are
+available in Kubernetes v1.25.
+{{< /note >}}
+
+When you enable `PodDisruptionsCondition` in your cluster, a dedicated Pod
```

Comment:
I think this should go in another page https://kubernetes.io/docs/concepts/workloads/pods/disruptions/

Reply:
updated with x-references between the pages.